starlite/openapi/path_item.py

```diff
@@ -44,9 +81,11 @@ def create_path_item(
                 request_body = create_request_body(
                     field=handler_fields["data"], generate_examples=create_examples, plugins=plugins
                 )
+
+            tags, security = extract_layered_values(route_handler)
             operation = Operation(
                 operationId=route_handler.operation_id or handler_name,
-                tags=extract_tags_from_route_handler(route_handler),
+                tags=list(set(tags)) if tags else None,
```

Comment:
```suggestion
                tags=tags,
```
The ternary and deduplication is done in return from `extract_layered_values()` by the looks.